### DISCUSSION TITLE
Brocken Pipe

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#112] [Q&A]
[Error] LyricExtractor.bash: Error: writing output failed: Broken pipe 
[Error] ArtworkExtractor.bash: Error: writing output failed: Broken pipe 
[Error] BeetsTagger.bash: Error: writing output failed: Broken pipe 

Just wondering how I might rectify these errors.

**Comment by RandomNinjaAtk** [comment]
#87

**Reply by thedinz** [reply]
So if understanding, it is benign? it produces an error line but it is in fact working.
